### PR TITLE
session push documentation

### PR DESCRIPTION
**Pull request by chistel**

### CODE REVIEW

**Comment by lonnieezell** [comment]
I don't believe this method exists currently. If it does I missed it just now.

**Comment by chistel** [comment]
you mean you thought it was already available??

**Comment by lonnieezell** [comment]
No, my point was that the function didn't exist. At the time, I hadn't seen your other PR for the actual function. I was confused. :)

File: user_guide_src/source/libraries/sessions.rst [hunk 1]
```diff
@@ -210,7 +210,14 @@ If you want to verify that a session value exists, simply check with
...
 
 	$session->has('some_name');
+	
+Pushing new value to session data
+=====================
+
+The push method is used to push a new value onto a session value that is an array. 
+For instance, if the 'hobbies' key contains an array of hobbies, you can add a new value onto the array like so::
 
+$session->push('hobbies', ['sport'=>'tennis']);
```
Comment (natanfelles) [inline user_guide_src/source/libraries/sessions.rst]: Something say me that this need a tab indentation and a line blank below.
And the title above need some `=`.

**Pull request merged**